packages/remix-react/components.tsx

```diff
@@ -719,7 +719,7 @@ export function Meta() {
         }
 
         if (name === "title") {
-          return <title key="title">{value}</title>;
+          return <title key="title">{value as string}</title>;
```

Comment:
nitpick, but i think i prefer this here instead of casting

```suggestion
          return <title key="title">{String(value)}</title>;
```